### PR TITLE
Add tests for core and hydrogens module

### PR DESCRIPTION
**Pull request by HubLot**
I added tests for the hydrogen module (I also reorganized the function inside this module for better clarity).

I started to add tests also to core module but it's more complicated since it requires a lot of initialization.
I'm not finished yet so don't merge it now. I opened this PR in case of some discussion is needed.

### CODE REVIEW

**Comment by HubLot** [comment]
Okay, I think I'm done for the hydrogens and core modules.
I'll do another PR for writers and cli modules.

BTW, Tests will have to be modified once we kept only one file for the ouput. A bit of refactoring on the compare files sections will not hurt either.

File: tests/test_core.py [hunk 1]
```diff
@@ -0,0 +1,327 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+
+
```
Comment (pierrepo) [inline tests/test_core.py:4]: These lines are not required anymore.

File: tests/test_dics.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+
+
```
Comment (pierrepo) [inline tests/test_dics.py:4]: These lines are not required anymore

File: tests/test_hydrogens.py [hunk 3]
```diff
@@ -0,0 +1,156 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+
+
```
Comment (pierrepo) [inline tests/test_hydrogens.py:4]: These lines are not required anymore

**Review by pierrepo — COMMENTED** [review]
Could you please lines
```
#! /usr/bin/env python
# -*- coding: utf-8 -*-


```
at the beginning of the file. They are not required anymore.

**Comment by HubLot** [comment]
done!

**Comment by pierrepo** [comment]
Thanks @HubLot This looks good to me!



@patrickfuchs would you merge this PR?

**Comment by patrickfuchs** [comment]
Yeaaaaaah :satisfied:

**Review by patrickfuchs — APPROVED** [review]

**Pull request merged**